bothub/api/v2/repository/views.py

```diff
@@ -715,21 +719,55 @@ def evaluate(self, request, **kwargs):
         user_authorization = repository.get_user_authorization(request.user)
         if not user_authorization.can_write:
             raise PermissionDenied()
-        serializer = EvaluateSerializer(data=request.data)  # pragma: no cover
-        serializer.is_valid(raise_exception=True)  # pragma: no cover
-
-        try:
-            request = repository.request_nlp_manual_evaluate(  # pragma: no cover
-                user_authorization, serializer.data
-            )
-        except DjangoValidationError as e:
-            raise APIException(e.message, code=400)
+        data = request.data
+        response = []
+        version_languages = RepositoryVersionLanguage.objects.filter(repository_version__pk=data.get("repository_version"))
+        for version_language in version_languages:
+            if not repository.have_at_least_one_test_phrase_registered(version_language.language):
+                continue
+            if "language" in data:
+                data["language"] = version_language.language
+            else:
+                data.update({"language": version_language.language})
+            serializer = EvaluateSerializer(data=data)  # pragma: no cover
+            serializer.is_valid(raise_exception=True)  # pragma: no cover
 
-        if request.status_code != status.HTTP_200_OK:  # pragma: no cover
-            raise APIException(
-                {"status_code": request.status_code}, code=request.status_code
-            )  # pragma: no cover
-        return Response(request.json())  # pragma: no cover
+            try:
+                nlp_request = repository.request_nlp_manual_evaluate(  # pragma: no cover
+                    user_authorization, serializer.data
+                )
+            except DjangoValidationError as e:
+                raise APIException(e.message, code=400)
+
+            if nlp_request.status_code != status.HTTP_200_OK:  # pragma: no cover
+                raise APIException(
+                    {"status_code": nlp_request.status_code}, code=nlp_request.status_code
+                )  # pragma: no cover
+            
```

Comment:
this line contains whitespace:

for more details: https://www.flake8rules.com/rules/W293.html